images/macos/provision/core/codeql-bundle.sh

```diff
@@ -2,8 +2,8 @@
 source ~/utils/utils.sh
 
 # Retrieve the CLI version of the latest CodeQL bundle.
-base_url="$(curl -fsSL https://raw.githubusercontent.com/github/codeql-action/v2/src/defaults.json)"
-bundle_version="$(echo "$base_url" | jq -r '.cliVersion')"
+download_with_retries https://raw.githubusercontent.com/github/codeql-action/v2/src/defaults.json "/tmp" "codeql-defaults.json"
+bundle_version="$(cat /tmp/codeql-defaults.json | jq -r '.cliVersion')"
```

Comment:
We can simplify this:
```suggestion
bundle_version="$(jq -r '.cliVersion' /tmp/codeql-defaults.json)"
```